recipe/meta.yaml

```diff
@@ -1,55 +1,77 @@
 {% set name = "pikepdf" %}
-{% set version = "5.6.1" %}
+{% set version = "9.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 03fb70ff4fada9cba864d04033f4166b5bf7fa85c90cdc05afd0ab87c9ccaa1a
+  sha256: 0108c063bc56dc2dbfc87f20533a728342a938f4c85e39773866b71255aa8388
+  patches:
+    - 0001-update-paths.patch
+    - 0002-fix-test.patch
+    - 0003-do-not-require-deprecated.patch
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win or py<37]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # skip s390x because of missing qpdf
+  skip: true  # [py<38 or s390x]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
-    - setuptools_scm >=4.1
-    - wheel >=0.35
-    - setuptools_scm_git_archive
-    - pybind11 >=2.6.0,< 3
-    - qpdf >=10.6.2
+    - setuptools_scm 8.1.0
+    - wheel
+    - pybind11 2.13.6
+    - qpdf 11.9.1
```

Comment:
[qpdf >=11.5.0](https://github.com/pikepdf/pikepdf/blob/v9.4.2/pyproject.toml#L89)?
There's the preferred version which you cite but it does give a min version.  And perhaps yours should be `>=`?  

In fact, is it even needed in `host` at all, I just ran a build without it...  It doesn't vendor it, does it?